src/test/java/ui/tests/RegistrationTest.java

```diff
@@ -0,0 +1,82 @@
+package ui.tests;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import pageobjects.AccountCreationPage;
+import pageobjects.AccountPage;
+import pageobjects.AuthenticationPage;
+import pageobjects.MainPage;
+import ui.ConfigProperties;
+import ui.UserAccount;
+import ui.UserAccountRegistrationForm;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+public class RegistrationTest {
+    public static WebDriver driver;
+    public static MainPage mainPage;
+    public static AuthenticationPage authenticationPage;
+    public static AccountCreationPage accountCreationPage;
+    public static AccountPage accountPage;
+    private static final Logger logger = Logger.getLogger(RegistrationTest.class.getName());
+
+    @BeforeClass
+    public static void setup() {
+        logger.info("Setup process started");
+        System.setProperty("webdriver.chrome.driver", ConfigProperties.getProperty("chromedriver"));
+        logger.info("ChromeDriver initiated");
```

Comment:
it just gets initiated next string :)

Reply:
sorry, I didn't get

Reply:
line 35: write to log: "driver initiated"
line 36: actual line when the driver gets initiated :D 

If the initialization fails, log will look a bit odd:
```
- driver initiated
- oops, it's actually failed
```

Reply:
Updated